pkg/workload/schemachange/operation_generator.go

```diff
@@ -2671,6 +2671,24 @@ func makeOpStmt(queryType opStmtType) *opStmt {
 	}
 }
 
+// opStmtFromTree constructs an operation from the provide tree.Statement.
+func newOpStmt(stmt tree.Statement, codes codesWithConditions) *opStmt {
+	queryType := OpStmtDDL
+	if stmt.StatementType() != tree.TypeDDL {
+		queryType = OpStmtDML
+	}
```

Comment:
What if "stmt.Type" is neither DDL nor DML? Should we be paranoid of such possibility?

Reply:
Switch to a switch and panicked in the default case.